pyaedt/modeler/cad/Primitives.py

```diff
@@ -4607,6 +4607,123 @@ def get_vertices_of_line(self, assignment):
 
         return position_list
 
+    @pyaedt_function_handler(object_list="assignment_to_export", removed_objects="assignment_to_remove")
+    def export_3d_model(
+        self,
+        file_name="",
+        file_path="",
+        file_format=".step",
+        assignment_to_export=None,
+        assignment_to_remove=None,
+        major_version=-1,
+        minor_version=-1,
+        **kwargs  # fmt: skip
+    ):
+        """Export the 3D model.
+
+        Parameters
+        ----------
+        file_name : str, optional
+            Name of the file.
+        file_path : str, optional
+            Path for the file.
+        file_format : str, optional
+            Format of the file. The default is ``".step"``.
+        assignment_to_export : list, optional
+            List of objects to export. The default is ``None``.
+        assignment_to_remove : list, optional
+            List of objects to remove. The default is ``None``.
+        major_version : int, optional
+            File format major version. Default is -1.
+        minor_version : int, optional
+            File format major version. Default is -1.
```

Comment:
```suggestion
            File format major version. The default is -1.
        minor_version : int, optional
            File format major version. The default is -1.
```
Can these last two descriptions be clearer?